pandas/tests/indexing/test_loc.py

```diff
@@ -346,6 +346,16 @@ def test_loc_index(self):
         result = df.loc[pd.array(mask, dtype="boolean")]
         tm.assert_frame_equal(result, expected)
 
+    def test_loc_copy_vs_view(self):
```

Comment:
hmm I am almost sure we have other tests about this, can you check and co-locate

Reply:
Yes you're right. This one
https://github.com/pandas-dev/pandas/blob/f5b2e5a9a2431165b1e8002219c5d3962d7bc920/pandas/tests/indexing/test_loc.py#L872
is very similar so moved it there